src/DataHandling/Logs/DataDogAudit.php

```diff
@@ -435,8 +435,8 @@ protected function filterFinalResult(array $diffArray)
         ) {
             // insert call missing, incomplete log => update placeholder
             $diffArray[self::UNKNOWN_VERSION_CHANGE] = array(
-            'changes' => array('  ' => 'unknown document versions before'),
-            'savedBy' => 'UNKNOWN USER',
+            'changes' => array('  ' => $this->translator->translate('label.unknown.versions-before')),
+            'savedBy' => $this->translator->translate('label.unknown.user'),
```

Comment:
What is the default translation domain?

Reply:
It is messages, so the file below has the wrong name. Or the domain must be set.

Reply:
according to [best practice](https://symfony.com/doc/current/bundles/best_practices.html#translation-files) the translation domain should be the bundle name (cube_common or cubetools_cube_common, whatever the name is).